server/routes/visit.test.ts

```diff
@@ -484,7 +484,14 @@ describe('/visit/:reference', () => {
 
     it('should redirect to /visit/:reference if selected establishment does not match prison for which visit booked', () => {
       app = appWithAllRoutes({
-        services: { auditService, supportedPrisonsService, visitService, visitSessionsService },
+        services: {
```

Comment:
updated this test because although its not required, the test would be unable to pass even if the data on 496 was correct